src/nginx-header.tmpl

```diff
@@ -39,25 +39,26 @@ http {
             return 503;
         }
     }
-    {{if and .LeaderVHost .Leader.Endpoint}}
-    upstream {{.LeaderVHost}} {
-        server {{.Leader.Host}}:{{.Leader.Port}};
+    
+    {{if and .Namespaces.stage1.LeaderVHost .Namespaces.stage1.Leader.Endpoint}}
```

Comment:
This will lead to very large breaking change. Put in defaults for namespace if possible

Reply:
Done